### PR TITLE
Use cake scripts from separate repo

### PR DESCRIPTION
**Pull request by fsobolev**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "CakeScripts"]
+	path = CakeScripts
```
Comment (nlogozzo) [inline .gitmodules:2]: Is this supposed to be indented here?

Comment (fsobolev) [reply .gitmodules:2]: idk, I didn't modify it, only ran `git submodule add...`

**Comment by nlogozzo** [comment]
CI needs fixing...

**Comment by fsobolev** [comment]
> CI needs fixing...

Looks like it doesn't load submodule 🤔

File: .github/workflows/flatpak.yml [hunk 2]
```diff
@@ -25,6 +25,8 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v3.5.0
+        with:
```
Comment (nlogozzo) [inline .github/workflows/flatpak.yml:28]: Should be added to snap workflow too, no?

Comment (fsobolev) [reply .github/workflows/flatpak.yml:28]: There's no snap workflow in this repo. But in other repos I don't think we need to change snap workflow, snapcraft loads submodules by default

Comment (nlogozzo) [reply .github/workflows/flatpak.yml:28]: > There's no snap workflow in this repo.

I'm dumb 😂😂

**Review by nlogozzo — APPROVED** [review]

**Pull request merged**